Render/RenderSettings.as

```diff
@@ -41,6 +42,14 @@ void RenderSettingsCustomization(){
 
     showPb = UI::Checkbox("Show personal best", showPb);
 
+    UI::BeginDisabled(!useExternalAPI);
+
+    UI::Text("\n\tPercentage ranking");
+
+    showPercentage = UI::Checkbox("Show percentage column (requires External API)", showPercentage && useExternalAPI);
+
+    UI::EndDisabled();
+
```

Comment:
Nice addition, i'll probably reuse that for exisiting/future features that requires the API